sys/net/gnrc/network_layer/ipv6/nib/_nib-slaac.c

```diff
@@ -162,10 +162,9 @@ void _remove_tentative_addr(gnrc_netif_t *netif, const ipv6_addr_t *addr)
         /* Cannot use target address as personal address and can
          * not change hardware address to retry SLAAC => use purely
          * DHCPv6 instead */
-        /* TODO: implement IA_NA for DHCPv6 */
-        /* then => tgt_netif->aac_mode |= GNRC_NETIF_AAC_DHCP; */
-        DEBUG("nib: would set interface %i to DHCPv6, "
-              "but is not implemented yet", netif->pid);
+
+        /* TODO: How will this flag be used? */
```

Comment:
IMHO the DHCPv6 client should check if

1. The flag is set before requesting `IA_NA` records for this interface,
2. Use the flag on start-up (in application space) to decide weather `dhcpv6_client_req_ia_na()` should be called for that interface.